docs/transparent-validation/rte/v1.1.0/specification.md

```diff
@@ -166,6 +166,34 @@ connector and can be enabled/disabled on demand:
 | 0 - low                 | disabled (by default)         |
 | 1 - high                | enabled                       |
 
+* GPIO states during flashing:
+
+1. Set proper SPI Vcc Voltage:
+
+    `echo 0 > /sys/class/gpio/gpio405/value` - sets SPI Vcc to 1.8V
+
+    Or when flash chip operates on 3.3V:
+
+    `echo 1 > /sys/class/gpio/gpio405/value` - sets SPI Vcc to 3.3V
+
+1. Enable SPI Vcc:
+
+    `echo 1 > /sys/class/gpio/gpio406/value`
+
+1. Enable SPI signals:
+
+    `echo 1 > /sys/class/gpio/gpio404/value`
+
+1. Flash SPI chip.
```

Comment:
Give the full command `flashrom -p spidev... -w <filename>`

Reply:
Done